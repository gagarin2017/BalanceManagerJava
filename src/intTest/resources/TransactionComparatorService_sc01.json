{
    "remoteAccountTxsData": [{
            "account": "Bank acc",
            "description": "description",
            "category": "Groceries",
            "amount": "-15.11",
            "clr": "R",
            "tag": "tag",
            "date": "02/05/2021",
            "memo": "memo"
        }, {
            "account": "Bank acc",
            "description": "description",
            "category": "Home insurance",
            "amount": "-22.31",
            "clr": "R",
            "tag": "tag",
            "date": "02/05/2021",
            "memo": "memo"
        }, {
            "account": "Bank acc",
            "description": "one month payment ",
            "category": "Car insurance",
            "amount": "-38.38",
            "clr": "R",
            "tag": "tag",
            "date": "31/05/2021",
            "memo": "memo"
        }, {
            "account": "Bank acc",
            "description": "gp visit",
            "category": "doctor",
            "amount": "-65.00",
            "clr": "R",
            "tag": "tag",
            "date": "31/05/2021",
            "memo": "memo"
        }
    ],
    "localAccountTxsData": [{
            "date": "02/05/2021",
            "account": "Bank acc",
            "description": "description",
            "memo": "memo",
            "category": "Groceries",
            "tag": "tag",
            "clr": "R",
            "amount": "-15.11"
        }, {
            "account": "Bank acc",
            "description": "description",
            "category": "Home insurance",
            "amount": "-22.31",
            "clr": "R",
            "tag": "tag",
            "date": "02/05/2021",
            "memo": "memo"
        }, {
            "account": "Bank acc",
            "description": "one month payment ",
            "category": "Car insurance",
            "amount": "-38.38",
            "clr": "R",
            "tag": "tag",
            "date": "31/05/2021",
            "memo": "memo"
        }, {
            "account": "Bank acc",
            "description": "gp visit",
            "category": "doctor",
            "amount": "-65.00",
            "clr": "R",
            "tag": "tag",
            "date": "31/05/2021",
            "memo": "memo"
        }
    ],
    "startingBalance": 15.12
}