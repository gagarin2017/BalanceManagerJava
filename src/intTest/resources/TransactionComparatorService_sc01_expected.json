{
    "remoteTransactions": [{
            "date": "02/05/2021",
            "transactions": [{
                    "description": "description",
                    "memo": "memo",
                    "clr": "R",
                    "tag": "tag",
                    "debitAmount": -15.11,
                    "creditAmount": 0,
                    "amountAsNumber": 0,
                    "amountNumberSignedString": "-",
                    "reconsiled": false,
                    "remote": false,
                    "date": "02/05/2021",
                    "account": "Bank acc",
                    "category": "Groceries"
                }, {
                    "description": "description",
                    "memo": "memo",
                    "clr": "R",
                    "tag": "tag",
                    "debitAmount": -22.31,
                    "creditAmount": 0,
                    "amountAsNumber": 0,
                    "amountNumberSignedString": "-",
                    "reconsiled": false,
                    "remote": false,
                    "date": "02/05/2021",
                    "account": "Bank acc",
                    "category": "Home insurance"
                }
            ],
            "total": 37.42
        }, {
            "date": "31/05/2021",
            "transactions": [{
                    "description": "one month payment ",
                    "memo": "memo",
                    "clr": "R",
                    "tag": "tag",
                    "debitAmount": -38.38,
                    "creditAmount": 0,
                    "amountAsNumber": 0,
                    "amountNumberSignedString": "-",
                    "reconsiled": false,
                    "remote": false,
                    "date": "31/05/2021",
                    "account": "Bank acc",
                    "category": "Car insurance"
                }, {
                    "description": "gp visit",
                    "memo": "memo",
                    "clr": "R",
                    "tag": "tag",
                    "debitAmount": -65.00,
                    "creditAmount": 0,
                    "amountAsNumber": 0,
                    "amountNumberSignedString": "-",
                    "reconsiled": false,
                    "remote": false,
                    "date": "31/05/2021",
                    "account": "Bank acc",
                    "category": "doctor"
                }
            ],
            "total": 103.38
        }
    ],
    "localTransactions": [{
            "date": "02/05/2021",
            "transactions": [{
                    "description": "description",
                    "memo": "memo",
                    "clr": "R",
                    "tag": "tag",
                    "debitAmount": -15.11,
                    "creditAmount": 0,
                    "amountAsNumber": 0,
                    "amountNumberSignedString": "-",
                    "reconsiled": false,
                    "remote": false,
                    "date": "02/05/2021",
                    "account": "Bank acc",
                    "category": "Groceries"
                }, {
                    "description": "description",
                    "memo": "memo",
                    "clr": "R",
                    "tag": "tag",
                    "debitAmount": -22.31,
                    "creditAmount": 0,
                    "amountAsNumber": 0,
                    "amountNumberSignedString": "-",
                    "reconsiled": false,
                    "remote": false,
                    "date": "02/05/2021",
                    "account": "Bank acc",
                    "category": "Home insurance"
                }
            ],
            "total": 37.42
        }, {
            "date": "31/05/2021",
            "transactions": [{
                    "description": "one month payment ",
                    "memo": "memo",
                    "clr": "R",
                    "tag": "tag",
                    "debitAmount": -38.38,
                    "creditAmount": 0,
                    "amountAsNumber": 0,
                    "amountNumberSignedString": "-",
                    "reconsiled": false,
                    "remote": false,
                    "date": "31/05/2021",
                    "account": "Bank acc",
                    "category": "Car insurance"
                }, {
                    "description": "gp visit",
                    "memo": "memo",
                    "clr": "R",
                    "tag": "tag",
                    "debitAmount": -65.00,
                    "creditAmount": 0,
                    "amountAsNumber": 0,
                    "amountNumberSignedString": "-",
                    "reconsiled": false,
                    "remote": false,
                    "date": "31/05/2021",
                    "account": "Bank acc",
                    "category": "doctor"
                }
            ],
            "total": 103.38
        }
    ],
    "missingTransactions": {},
    "errors": [],
    "infoMessages": ["The considered transaction data range for Remote: 02/05/2021 - 31/05/2021. Size: 2 days with transactions.", "The considered transaction data range for Local subset: 02/05/2021 - 31/05/2021. Size: 2 days with transactions.", "Comparing transactions for each date Remote [2 days with transactions] vs Local [2 days with transactions]."]
}
